### PR TITLE
AdminChevronFilteringTest refactored + createEntity method created

### PR DESCRIPTION
**Pull request by Vadym11**

### CODE REVIEW

**Review by guram21 — APPROVED** [review]

File: src/test/java/utils/ProjectUtils.java [hunk 1]
```diff
@@ -78,6 +79,14 @@ public static AdminFieldsPage createAllFields(WebDriver driver, String[] fieldTy
...
     }
 
+    public static AdminFieldsPage createEntity(String entityName, WebDriver driver) {
+
+        return new EntityTablePage(driver)
+                .clickConfiguration()
+                .clickNewEntity()
+                .createEntity(entityName);
+    }
+
```
Comment (oooleks) [inline src/test/java/utils/ProjectUtils.java:89]: Replace this method to class ProjectUtils.Portal

File: src/test/java/utils/ProjectUtils.java [hunk 2]
```diff
@@ -78,6 +79,14 @@ public static AdminFieldsPage createAllFields(WebDriver driver, String[] fieldTy
         }
     }
 
+    public static AdminFieldsPage createEntity(String entityName, WebDriver driver) {
```
Comment (oooleks) [inline src/test/java/utils/ProjectUtils.java:82]: To be consistent with project method change attribute order to have WebDriver in a first place

File: src/test/java/utils/ProjectUtils.java (hunk 1 above)
Comment (Vadym11) [reply src/test/java/utils/ProjectUtils.java]: done

File: src/test/java/utils/ProjectUtils.java (hunk 2 above)
Comment (Vadym11) [reply src/test/java/utils/ProjectUtils.java]: done

**Review by Burdishka — APPROVED** [review]

File: src/test/java/utils/ProjectUtils.java [hunk 3]
```diff
@@ -78,6 +79,14 @@ public static AdminFieldsPage createAllFields(WebDriver driver, String[] fieldTy
         }
     }
 
+        public static AdminFieldsPage createEntity(WebDriver driver, String entityName) {
```
Comment (oooleks) [inline src/test/java/utils/ProjectUtils.java:82]: The method has not been replaced to Portal class

**Review by Stan575 — APPROVED** [review]

File: src/test/java/utils/ProjectUtils.java (hunk 3 above)
Comment (Vadym11) [reply src/test/java/utils/ProjectUtils.java:82]: Hasn’t it? Then I need more details cuz I can’t understand where to move it. Could you please specify?

Comment (oooleks) [reply src/test/java/utils/ProjectUtils.java:82]: Class ProjectUtils has 2 classes: Portal and Entity, so place your method into Portal class but not in ProjectUtils class root location as your method belongs to Portal(Admin) functionality.

**Pull request merged**